web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -56,7 +56,7 @@
               :to="{ name: 'fileBrowser', query: { location: rootDirectory } }"
               style="text-decoration: none;"
             >
-              <v-icon>mdi-home</v-icon>
+              {{ identifier }}
```

Comment:
there might be some css tune up to do to harmonize spacing since now identifier is placed without spacing like present for other components of the breadcrumbs:

![image](https://user-images.githubusercontent.com/39889/117368607-bf6cc680-ae91-11eb-93ac-90571f683e3e.png)

Reply:
```suggestion
              <div class="mx-2">
                {{ identifier }}
              </div>
```
Might fix the spacing, although I can't really test how it renders in a suggestion.

Reply:
For some reason you can't put a `<div>` element within the `<router-link>` tag, but I instead added the `mx-2` class to the `<router-link>` itself and that seems to have worked (see 6135f23fbe462cf49f71f559d7413e7d75d0e356).